win32/src/win32inet.i

```diff
@@ -16,6 +16,13 @@
 #undef PyHANDLE // undef earlier define, so we are back to the class.
 #include "pywinobjects.h"
 
+#ifndef _MSC_VER
```

Comment:
Try not to add these definitions in projects. These should go into mingw-w64 project's header file. I will be adding those in that project.